.github/workflows/codegen-ci.yml

```diff
@@ -80,15 +75,36 @@ jobs:
       - name: Mock looker.ini
         run: |
           echo "[Looker]" > looker.ini
-          echo "base_url=https://self-signed.looker.com:19999" >> looker.ini
-          echo "client_id=id" >> looker.ini
-          echo "client_secret=secret" >> looker.ini
-          echo "verify_ssl=False" >> looker.ini
-          echo "timeout=31" >> looker.ini
-          echo "[Looker31]" >> looker.ini
-          echo "base_url=https://self-signed.looker.com:19999" >> looker.ini
-          echo "verify_ssl=False" >> looker.ini
+          echo "base_url=https://localhost:20000" >> looker.ini
+          echo "verify_ssl=false" >> looker.ini
```

Comment:
our SDKs **should** treat `false` and `False` identically FWIW

Reply:
I could almost swear that the path it takes in `fetchSpec` (where you just `ApiConfig` it) it was failing with `False` and then passed with `false`.